build.sh

```diff
@@ -12,10 +12,25 @@ function build() {
         -o ${plugin_name}
 }
 
+function buildstatic() {
+    local version=$1
+    local platform=$2
+    local arch=$3
+    local plugin_name=$4
+
+    echo calling to build static for $platform $arch
+    CGO_ENABLED=0 GOOS=$platform GOARCH=$arch go build -a -tags netgo \
+        -ldflags "-w -extldflags \"-static\" -X main.Version=${version}" \
```

Comment:
Just wondering - which binaries which you require are missing in the final produced build?

Reply:
I was building only for all the platforms. All build were succesful atleast on my system.